### PR TITLE
[t] test empty APICAST_LOG_LEVEL

### PR DESCRIPTION
**Pull request by mikz**
Regression test for https://github.com/3scale/apicast/pull/868

### CODE REVIEW

File: t/apicast-bin.t [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+use lib 't';
+use Test::APIcast::Blackbox 'no_plan';
+
+# Can't run twice because one of the test checks the contents of the cache, and
```
Comment (davidor) [inline t/apicast-bin.t]: :eyes:

File: t/apicast-bin.t [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# those change between runs (cache miss in first run, cache hit in second).
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: Empty APICAST_LOG_LEVEL is ignored.
```
Comment (davidor) [inline t/apicast-bin.t]: Should we say that the default applies rather than it is ignored?

File: t/apicast-bin.t (hunk 1 above)
Comment (mikz) [reply t/apicast-bin.t]: :-D  Copy 🍝

File: t/apicast-bin.t (hunk 2 above)
Comment (mikz) [reply t/apicast-bin.t]: Then we should test the default and that's harder than just verifying it works.

Comment (mikz) [reply t/apicast-bin.t]: I'll check it to "does not crash".

Comment (davidor) [reply t/apicast-bin.t]: :+1:

**Pull request merged**